libs/api/domains/health-insurance/src/lib/accident-notification.service.ts

```
@@ -56,42 +55,34 @@
     if (!accidentStatus) return null
     return {
-      numberIHI: accidentStatus.requestId,
+      numberIHI: accidentStatus.numberIHI,
       status: accidentStatus.status ? mapStatus(accidentStatus.status) : '',
       receivedAttachments: accidentStatus.attachments
         ?.map((x) => ({
-          [mapAttachmentType(x.type)]: !!x.received,
+          [mapAttachmentType(x.attachmentType)]: !!x.isReceived,
         }))
         .reduce(
           (prev, curr) => ({ ...prev, ...curr }),
```

Biome

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)